README.md

```diff
@@ -34,7 +102,7 @@ If you have what looks like a bug, please use the [GitHub issue tracking system]
 
 ### Contributing code to `osv-scanner`
 
-See [CONTRIBUTING.md](CONTRIBUTING.md) for documentation on how to contribute code.
+See [CONTRIBUTING.md](http://CONTRIBUTING.md) for documentation on how to contribute code.
```

Comment:
This should not be linking to the website http://contributing.md - the original seems to link correctly to the `CONTRIBUTING.md` file in the repo.